ui/src/features/project/project-details/nodes/repo-node.tsx

```diff
@@ -43,20 +44,55 @@ export const RepoNode = ({ nodeData, height }: Props) => (
         <RepoNodeBody label='Registry URL' value={nodeData.data.registryUrl} />
       )}
       {nodeData.type === NodeType.WAREHOUSE && (
-        <div className='text-center font-semibold mt-4'>{nodeData.data as string}</div>
+        <div className='text-center font-semibold mt-2'>
+          {(nodeData?.data?.spec?.subscriptions || []).map((sub, i) => {
+            return (
+              <div key={`${nodeData.data.metadata?.name}-${i}`}>
+                {sub.chart && (
+                  <RepoNodeBody
+                    label='Registry URL'
+                    value={sub.chart.registryUrl}
```

Comment:
<img width="90" alt="image" src="https://github.com/akuity/kargo/assets/4256585/fb29c68c-7b58-4f81-a846-60b33affa7d9">

maybe the chart.name is sufficient, doesn't the subsciption target a specific chart in the registry? Would assume multiple entries in the warehouse if you got 2 subscriptions on the same chartRegistry